src/nanoemoji/maximum_color.py

```diff
@@ -116,13 +151,62 @@ def _write_preamble(nw: NinjaWriter, input_font: Path):
     module_rule(
         nw,
         "glue_together",
-        f"--color_table COLR --target_font {rel_build(input_font)} --donor_font $in --output_file $out",
+        f"--color_table COLR --target_font {input_font} --donor_font $in --output_file $out",
         rule_name="copy_colr_from_svg2colr",
     )
     nw.newline()
 
+    module_rule(
+        nw,
+        "glue_together",
+        f"--color_table SVG --target_font {input_font} --donor_font $in --output_file $out",
+        rule_name="copy_svg_from_colr2svg",
+    )
+    nw.newline()
+
+
+def _generate_svg_from_colr(nw: NinjaWriter, input_font: Path, font: ttLib.TTFont):
+    # generate svgs
+    svg_files = [
+        rel_build(svg_generate_dir() / f"{gid:05d}.svg") for gid in colr_glyphs(font)
+    ]
+    nw.build(svg_files, "generate_svgs_from_colr", input_font)
+    nw.newline()
+
+    # picosvg them
```

Comment:
i think we could get rid of this step because the SVGs extracted by colr_to_svg.py are already (or should already be) "picosvg"

Reply:
Nevermind actually, we do emit g elements which are not pico. Let's keep for now